### DISCUSSION TITLE
Improve "Two-column category menu" addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#4081] [Ideas]
I would love to have some more options in this very useful addon (for those ones that are used to the old organization of block categories in Scratch 1.4 and 2.0):
- hide "add extension" button
- reduce the vertical space between categories

**Comment by TheColaber** [comment]
Maybe these are things that should be included in https://github.com/ScratchAddons/ScratchAddons/pull/4019
